docs/Spatial.md

```diff
@@ -1,6 +1,99 @@
 #Phidget Spatial Sensor
-More commonly known as an accelerometer.
+More commonly known as an accelerometer, the PhidgetSpatial library makes for intuitive and lightning fast development without any compromise. All spacial directions must be defined by user, for an in dept guide into this, [here is a great guide](http://www.phidgets.com/docs/1044_User_Guide). For a quick start into your Spatial project see this [basic Manager example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/spatial.js).
 
-[Basic spatial example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/spatial.js)
+##Methods
 
-ToDo : map out module and describe use
\ No newline at end of file
+The below information lists the parameters, events, and give you further information about your __Phidgets Spatial__. For low level development, [see here](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams).
```

Comment:
Please stop using low level. Please stop referencing  low level development as we discussed.